apps/sasquatch/src/main/java/com/microsoft/azure/mobile/sasquatch/activities/EventActivity.java

```diff
@@ -9,5 +12,6 @@
     @Override
     protected void trackLog(String name, Map<String, String> properties) {
         Analytics.trackEvent(name, properties);
+        Toast.makeText(getBaseContext(), R.string.description_event, Toast.LENGTH_SHORT).show();
```

Comment:
Use new texts for that, it's a bit weird to reuse that text when an event has just been sent.
Maybe use "Event '%1s' sent"